content/docs/100-get-started/400-offline-support.mdx

```diff
@@ -27,7 +27,92 @@ Below is a table of the TTL values of the on-device cache by content type.
 
 Content types that never expire such as images and videos will need to be cache busted manually if they are dynamic.
 
-[TODO: Content type table]
+<OptionsTable>
+  <OptionTitle>
+    ### `application/json`
+  </OptionTitle>
+  <OptionDescription>
+    Cached for 5 seconds
+  </OptionDescription>
+  <OptionTitle>
+    ### `application/pdf`
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `application/octet-stream`
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `binary/octet-stream`
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `image/jpeg`
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `image/gif`  
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `image/png`
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `image/bmp`  
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `image/webp`  
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `video/webm`
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `video/ogg`
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `video/mp4`   
+  </OptionTitle>
+  <OptionDescription>
+    Cached forever
+  </OptionDescription>
+  <OptionTitle>
+    ### `application/binary`  
+  </OptionTitle>
+  <OptionDescription>
+    Never cached
+  </OptionDescription>
+  <OptionTitle>
+    ### `~*mpegurl`
+  </OptionTitle>
+  <OptionDescription>
+    Never cached
+  </OptionDescription>
+</OptionsTable>
```

Comment:
The wrapping for the OptionsTable seems to be a bit buggy:

<img width="676" alt="image" src="https://user-images.githubusercontent.com/1163252/175380597-4af8a591-e4bd-41f2-9961-4c0140362b0f.png">


https://developer-docs-git-jschr-featcontent-raydiant.vercel.app/docs/get-started/offline-support#content-type-ttls